net/trojan-go/Portfile

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/p4gefau1t/trojan-go v0.8.2
+set git-commit      d051cf4c8852d708769ca1c4e514306a88da830b
```

Comment:
I usually don't bother with this, and instead set the embedded commit hash to `unknown`, but if you don't mind the hassle then good on you.